Documentation/trying-out-rkt.md

```diff
@@ -16,6 +16,10 @@ cd rkt-v1.8.0
 ./rkt help
 ```
 
+Like most applications, the easiest way to run rkt is to install it with your system's package manager, like apt on Debian or dnf on Fedora. If your distribution doesn't include a rkt package, or your operating system isn't Linux at all, running rkt in a Vagrant virtual machine can be nearly as simple. The instructions below start a virtual machine with rkt installed and ready to run.
```

Comment:
Point 1 is even more nuanced, as we just offer amd64 binaries here. We are getting [support for arm64](https://github.com/coreos/rkt/pull/2758) as a first class citizen arch, and Debian already builds rkt for way [many more](https://buildd.debian.org/status/package.php?p=rkt&suite=unstable) arches.